src/openforms/registrations/contrib/objects_api/templatetags/registrations/contrib/objects_api/json_tags.py

```diff
@@ -32,3 +37,10 @@ def json_summary(context):
         json_data = html_escape_json(json_data)
 
     return SafeString(json.dumps(json_data))
+
+
+@register.simple_tag
+def as_json(data: Any) -> SafeString:
```

Comment:
Just for the record - this template tag is off-limits for our own templates/developers, it only exists to accommodate user-supplied templates. It's a footgun.